java/arcs/core/host/BUILD

```diff
@@ -0,0 +1,14 @@
+load("//tools/build_defs/kotlin:rules.bzl", "kt_android_library", "kt_jvm_library")
+
+licenses(["notice"])
+
+package(default_visibility = ["//visibility:public"])
+
+kt_jvm_library(
```

Comment:
Currently restricted to JVM because of AutoService usage. I'll add a follow on CL that moves the ServiceLoader registry to a JVM specific directory, as I plan on adding Dagger2/Sting discovery versons of HostRegistry which should support multiplatform